frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/advanced-section.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Checkbox,
+  CheckboxProps,
+  ExpandableSection,
+  ExpandableSectionProps,
+  FormGroup,
+} from '@patternfly/react-core';
+import { DeploymentType } from '../../../../constants/create-storage-system';
+import { WizardDispatch, WizardState } from '../../reducer';
+import './backing-storage.scss';
+
+const RADIO_GROUP_NAME = 'deployment-type';
+
+export const AdvancedSection: React.FC<AdvancedSelectionProps> = ({
+  deployment,
+  isAdvancedOpen,
+  dispatch,
+}) => {
+  const { t } = useTranslation();
+
+  const handleSelection: CheckboxProps['onChange'] = (checked) => {
+    dispatch({
+      type: 'backingStorage/setDeployment',
+      payload: checked ? DeploymentType.MCG : DeploymentType.ALL,
+    });
+  };
+  const handleExpanadableToggling: ExpandableSectionProps['onToggle'] = (isExpanded: boolean) => {
+    dispatch({ type: 'backingStorage/setIsAdvancedOpen', payload: isExpanded });
+  };
+
+  return (
+    <ExpandableSection
+      toggleText={t('ceph-storage-plugin~Advanced')}
+      onToggle={handleExpanadableToggling}
+      isExpanded={isAdvancedOpen}
+    >
+      <FormGroup label={t('ceph-storage-plugin~Deployment type')} fieldId="deployment-type">
+        <Checkbox
+          label={t('ceph-storage-plugin~Multi cloud object gateway')}
+          description={t('ceph-storage-plugin~Object storage')}
```

Comment:
```suggestion
          description={t('ceph-storage-plugin~Checking this box will enable a Multi cloud gateway only deployment. Only object storage can be managed using this type of deployment')}
```

we should explain to users about this option

Reply:
Noting this for doc review